beacon-chain/core/helpers/slot_epoch.go

```diff
@@ -90,7 +90,7 @@ func SlotsSinceEpochStarts(slot uint64) uint64 {
 }
 
 // TimeShiftTolerance specifies the tolerance threshold for slots "from the future".
-const TimeShiftTolerance = 500 * time.Millisecond // ms
+const TimeShiftTolerance = 10 * time.Millisecond // ms
```

Comment:
hey can you change this back , I added a comment on this here: 
https://github.com/prysmaticlabs/prysm/pull/5127#discussion_r407788653

Reply:
kk